<?xml version="1.0"?>

<launch>
      <!-- Start Gazebo with wg world running in (max) realtime -->
    <include file="$(find hector_gazebo_worlds)/launch/rolling_landscape_120m.launch"/>

    <group ns="drone1">
        <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
            <arg name="tf_prefix" value="drone1" />
            <arg name="name" value="drone1" />
            <arg name="y" value="-1.0" />
            <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
            <arg name="controllers" value="
            controller/attitude
            controller/velocity
            controller/position
            "/>
        </include>
    </group>

    <group ns="drone2">
        <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
            <arg name="tf_prefix" value="drone2" />
            <arg name="name" value="drone2" />
            <arg name="y" value="1.0" />
            <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
            <arg name="controllers" value="
            controller/attitude
            controller/velocity
            controller/position
            "/>
        </include>
    </group>
    
    <node pkg="two_drones" name="hover_node" type="hover_node.cpp" respawn="true"/>

      <!-- Start rviz visualization with preset config -->
      <node pkg="rviz" type="rviz" name="rviz" args="-d $(find hector_quadrotor_demo)/rviz_cfg/outdoor_two_drone.rviz"/>

</launch>